components/server/README.md

```diff
@@ -10,315 +10,189 @@ API documentation can be retrieved via http://www.quality-time.example.org/api (
 
 ## Data model
 
-The [`datamodel.json`](src/data/datamodel.json) describes the domain model used by the application. It allows for a frontend that doesn't need to know about specific metrics and sources. Every time the server is started, the latest data model is imported into the database. The server [`data` package](src/data/datamodel.json) contains the `datamodel.json`.
+The data model package describes the domain model used by the application. It allows for a frontend that doesn't need to know about specific metrics and sources. On server start up, it checks whether the data model has changed, and if so, imports it into the database.
```

Comment:
The package is currently called "data" instead of "data model" since it also still contains the example reports. Maybe it would be a good idea to give the data model it's own package and make call the data package example_data?